zerver/migrations/0001_initial.py

```diff
@@ -256,7 +256,7 @@ class Migration(migrations.Migration):
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('name', models.CharField(db_index=True, max_length=60)),
                 ('invite_only', models.NullBooleanField(default=False)),
-                ('email_token', models.CharField(default=zerver.models.generate_email_token_for_stream, max_length=32)),
+                ('email_token', models.CharField(default=generate_email_token_for_stream, max_length=32)),
```

Comment:
So, this commits breaks lint temporarily; it'd be better to have this be squashed with the change that modifies the enforcement.

Reply:
I took care of doing the squash so I could merge.